src/Microsoft.TemplateEngine.Edge/Settings/TemplateCache.cs

```diff
@@ -173,25 +188,48 @@ private void ScanMountPointForTemplatesAndLangpacks(IMountPoint mountPoint, stri
                 {
                     AddTemplateToMemoryCache(template);
                 }
+
+                anythingFound |= templateList.Count > 0;
```

Comment:
anythingFound doesn't take into account the found langpacks. I think this should be:

anythingFound |= templateList.Count > 0 || localizationInfo.Count > 0